instrumentation/net/http/otelhttp/handler_test.go

```diff
@@ -796,63 +796,155 @@ func TestHandlerWithMetricAttributesFn(t *testing.T) {
+	)
+	newMetrics := []metricdata.Metrics{
+		{
+			Name:        "http.server.request.body.size",
+			Description: "Size of HTTP server request bodies.",
+			Unit:        "By",
+			Data: metricdata.Histogram[int64]{
+				Temporality: metricdata.CumulativeTemporality,
+				DataPoints:  []metricdata.HistogramDataPoint[int64]{{Attributes: newMetricAttrs}},
+			},
+		},
+		{
+			Name:        "http.server.response.body.size",
+			Description: "Size of HTTP server response bodies.",
+			Unit:        "By",
+			Data: metricdata.Histogram[int64]{
+				Temporality: metricdata.CumulativeTemporality,
+				DataPoints:  []metricdata.HistogramDataPoint[int64]{{Attributes: newMetricAttrs}},
+			},
+		},
+		{
+			Name:        "http.server.request.duration",
+			Description: "Duration of HTTP server requests.",
+			Unit:        "s",
+			Data: metricdata.Histogram[float64]{
+				Temporality: metricdata.CumulativeTemporality,
+				DataPoints:  []metricdata.HistogramDataPoint[float64]{{Attributes: newMetricAttrs}},
+			},
+		},
+	}
+	oldSpanAttrs := []attribute.KeyValue{
+		attribute.String("http.method", "GET"),
+		attribute.String("http.scheme", "http"),
+		attribute.String("net.host.name", "localhost"),
+		attribute.String("net.protocol.version", "1.1"),
+		attribute.String("http.target", "/"),
+		attribute.Int("http.status_code", 200),
+	}
+	oldMetricAttrs := attribute.NewSet(
+		attribute.String("http.method", "GET"),
+		attribute.String("http.scheme", "http"),
+		attribute.Int("http.status_code", 200),
+		attribute.String("net.host.name", "localhost"),
+		attribute.String("net.protocol.name", "http"),
+		attribute.String("net.protocol.version", "1.1"),
+	)
+	oldMetrics := []metricdata.Metrics{
+		{
+			Name:        "http.server.request.size",
+			Description: "Measures the size of HTTP request messages.",
+			Unit:        "By",
+			Data: metricdata.Sum[int64]{
+				Temporality: metricdata.CumulativeTemporality,
+				IsMonotonic: true,
+				DataPoints:  []metricdata.DataPoint[int64]{{Attributes: oldMetricAttrs}},
+			},
+		},
+		{
+			Name:        "http.server.response.size",
+			Description: "Measures the size of HTTP response messages.",
+			Unit:        "By",
+			Data: metricdata.Sum[int64]{
+				Temporality: metricdata.CumulativeTemporality,
+				IsMonotonic: true,
+				DataPoints:  []metricdata.DataPoint[int64]{{Attributes: oldMetricAttrs}},
+			},
+		},
+		{
+			Name:        "http.server.duration",
+			Description: "Measures the duration of inbound HTTP requests.",
+			Unit:        "ms",
+			Data: metricdata.Histogram[float64]{
+				Temporality: metricdata.CumulativeTemporality,
+				DataPoints:  []metricdata.HistogramDataPoint[float64]{{Attributes: oldMetricAttrs}},
+			},
+		},
+	}
 	tests := []struct {
 		name                       string
 		semConvStabilityOptInValue string
-		expected                   []attribute.KeyValue
+		expectedSpanAttributes     []attribute.KeyValue
+		expectedMetrics            metricdata.ScopeMetrics
```

Comment:
```suggestion
		wantSpanAttributes     []attribute.KeyValue
		wantMetrics            metricdata.ScopeMetrics
```

Small change (I know it wasn't like this before, but it's a convention we used that must have fallen through the cracks here).